### ISSUE TITLE
OrderedDict: builtins:AttributeError: No such attribute: get

### ISSUE THREAD

**Issue by petebrowne** [joeljames/py-shared-utils#2]
Currently, standard dict methods like `get`, `pop`, `setdefault`, etc are not available on `OrderedDict`

```python
>>> from pysharedutils.dicts import ObjectDict
>>> d = ObjectDict({'a': 'b'})
>>> d.a
'b'
>>> d.get('a')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.4/dist-packages/pysharedutils/dicts.py", line 84, in __getattr__
    'No such attribute: %s' % (name)
AttributeError: No such attribute: get
```

It'd be pretty simple to add - I'm just wondering if this makes sense or not? What do you think?

**Comment by joeljames** [comment]
Nice idea. It makes sense to add.

**Comment by petebrowne** [comment]
Fixed in #3
